src/tests/Dashboard/action/specificConceptAction.test.js

```diff
@@ -86,4 +86,24 @@ describe('Test suite for specific concepts actions', () => {
       expect(store.getActions()).toEqual(expectedActions);
     });
   });
+
+  it('should return an error message if request fails', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.reject({});
+    });
+
+    const expectedActions = [
+      { type: IS_FETCHING, payload: true },
+      { type: IS_FETCHING, payload: false },
+      { type: FETCH_CONCEPTS, payload: 'Request can\'t be made' },
+      { type: IS_FETCHING, payload: false },
+    ];
+
+    const store = mockStore({ payload: {} });
+
+    return store.dispatch(fetchConceptsActionTypes('malaria', 10, 1)).then(() => {
+      expect(store.getActions()).toEqual(expectedActions);
```

Comment:
How does this tell that an error message is returned?

Reply:
The expected actions above include one with a payload consisting an error message.